tests/dummy/app/controllers/application.js

```diff
@@ -3,6 +3,7 @@ import Ember from 'ember';
 export default Ember.Controller.extend({
   isDemoVisible: true,
   myProgress: 0.5,
+  isFirstSwitchOn: true,
```

Comment:
This seems unrelated and weird. It's weird because I remember doing this when I was working on the switch in one of my branches, but I also remember taking it out.

Reply:
It is unrelated, but it makes sure that the switch on the demo under "Active Switch" actually begins active.

Reply:
I see. 👌